dot-atom/keymap.cson

```diff
@@ -21,7 +21,12 @@
 # * https://atom.io/docs/latest/using-atom-basic-customization#customizing-key-bindings
 # * https://atom.io/docs/latest/behind-atom-keymaps-in-depth
 #
+# If you're having trouble with your keybindings not working, try the
+# Keybinding Resolver: Cmd+. on OS X and Ctrl+. on other platforms. See the
+# Debugging Guide for more information:
+# * https://atom.io/docs/v1.3.1/hacking-atom-debugging#check-the-keybindings
```

Comment:
Should this be https://atom.io/docs/latest/hacking-atom-debugging#check-the-keybindings instead?